google/cloud/security/inventory/pipelines/load_org_iam_policies_pipeline.py

```diff
@@ -24,58 +24,78 @@
 from google.cloud.security.common.gcp_api.cloud_resource_manager import CloudResourceManagerClient
```

Comment:
Modules should be imported, not Classes, see https://google.github.io/styleguide/pyguide.html#Imports.
Example:

from google.cloud.security.common.data_access import errors as data_access_errors

...

except (data_access_errors.ApiExecutionError, data_access_errors.MySQLError) as e:
...